optaplanner-operator/pom.xml

```diff
@@ -11,6 +11,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>optaplanner-operator</artifactId>
```

Comment:
Now it should be called `optaplanner-operator-parent` to be consistent with other parent modules.

Reply:
a60c0a97